synedrion/src/curve/arithmetic.rs

```diff
@@ -410,4 +546,31 @@ mod test {
         let s_from_le_bytes = Scalar::try_from_be_bytes(&le_bytes).expect("bytes are valid-ish");
         assert_ne!(s, s_from_le_bytes, "Using LE bytes should not work")
     }
+
+    #[test]
+    fn to_and_from_bytes_tiny32() {
```

Comment:
Looks like it could be just a generic function called with different `SchemeParams` types. 

Also not sure what this test is actually testing in the second part. Why all this business with inverse? It can actually create an invalid Scalar representation, if it reverses something like `01 FF FF ... FF`.

Reply:
I think this test should be removed. It's just a copy of the existing test above + tweaks and a different parametrization.